Sources/Xcore/SwiftUI/Components/WebView/WebView.swift

```diff
@@ -246,8 +256,14 @@ extension WebView {
         }
 
         func webView(_ webView: WKWebView, createWebViewWith configuration: WKWebViewConfiguration, for navigationAction: WKNavigationAction, windowFeatures: WKWindowFeatures) -> WKWebView? {
-            webView.load(navigationAction.request)
-            return nil
+            guard let url = navigationAction.request.url else {
```

Comment:
don't we actually want the request itself? since it can have other info like headers etc.